Sources/LayoutArrangement.swift

```diff
@@ -171,5 +177,16 @@ extension View {
             addSubview(view)
         }
     }
+
+    fileprivate func isChildOf(_ view: View) -> Bool {
```

Comment:
Please, check this:
https://developer.apple.com/documentation/uikit/uiview/1622521-isdescendant